cpp/src/table/table_device_view.cu

```diff
@@ -67,13 +66,13 @@ table_device_view_base<ColumnDeviceView, HostTableView>::table_device_view_base(
     // We need this pointer in order to pass it down when creating the
     // ColumnDeviceViews so the column can set the pointer(s) for any
     // of its child objects.
-    RMM_TRY(RMM_ALLOC(&_columns, views_size_bytes, stream));
-    ColumnDeviceView* d_column = _columns;
+    _descendant_storage = new rmm::device_buffer(views_size_bytes, stream);
```

Comment:
Where is this deleted?

Reply:
A deleter is declared in https://github.com/rapidsai/cudf/pull/4716/files#diff-e01f0e2e33cb3dd4ec3d73bbd9130577R82